ouroboros-consensus-cardano/tools/db-analyser/Analysis.hs

```diff
@@ -192,22 +188,28 @@ instance HasAnalysis blk => Show (TraceEvent blk) where
 -------------------------------------------------------------------------------}
 
 showSlotBlockNo :: forall blk. HasAnalysis blk => Analysis blk
-showSlotBlockNo AnalysisEnv { db, registry, initLedger, limit, tracer } =
-    processAll_ db registry GetHeader initLedger limit process
+showSlotBlockNo AnalysisEnv { db, initLedger, limit, tracer, ledgerDbFS, cfg } = do
+  doCheck <- onlyCheckNumBlocks limit
+  initLedger' <- initialLedger ledgerDbFS cfg initLedger
```

Comment:
`initLedger` and `initialLedger` seems to denote the same. Is `initialiseLedger` a more appropriate name for the latter?